codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspMethods.java

```diff
@@ -26,11 +26,12 @@
 public final class RtspMethods {
 
```

Comment:
please revert changes in this method as these are not needed for the functionality